src/views/Login.vue

```diff
@@ -0,0 +1,90 @@
+<template>
```

Comment:
We will need a Login component here, and then that Login component is reusable across places, we can embed the Login component in this view. Please separate the two.

Reply:
I don't understand what we need to do here. Separate which two components?

Reply:
This code should be inside one component `src/components/LoginCard.vue`
https://github.com/devlup-labs/gymkhana_frontend/blob/28d071c7b4a39af997e4099c89f015f74f5c67a4/src/views/Login.vue#L8-L59

And then you should import this inside `src/views/Login.vue`.

Reply:
okat got it

Reply:
please review